src/homepageExperience/components/steps/arduino/PrepareIde.tsx

```diff
@@ -0,0 +1,44 @@
+// Libraries
+import React from 'react'
+
+export const PrepareIde = () => {
+  return (
+    <>
+      <h1>Prepare Arduino IDE</h1>
+      <p>
+        The easiest way to get started with the InfluxDB Arduino client is with
+        the ESP8266 or ESP32 board.
+      </p>
+      <p>
+        If you haven't already, add the board you wish to use (ESP8266 or ESP32)
+        to the Arduino IDE by following these steps:
+      </p>
+      <p>For ESP8266:</p>
+      <ol style={{fontSize: '16px'}}>
+        <li>Open the Arduino Preferences (Arduino {'>'} Preferences) </li>
+        <li>
+          Look for "Additional Boards Manager URLs" input box and paste
+          "http://arduino.esp8266.com/stable/package_esp8266com_index.json" in
+          it.
+        </li>
+        <li>Click OK.</li>
+        <li>Then under Tools {'>'} Boards: , click on Boards Manager.</li>
```

Comment:
For these lines, can we make the style so it's not bold? it's subtly more bolder than the other text.